src/components/Modals/Send/Form.tsx

```diff
@@ -29,7 +26,7 @@ export type SendInput<T extends ChainId = ChainId> = {
   [SendFormFields.Input]: string
   [SendFormFields.Address]: string
   [SendFormFields.VanityAddress]: string
-  [SendFormFields.AccountId]: AccountSpecifier
+  [SendFormFields.AccountId]: string
```

Comment:
unintentional, will revert

Reply:
`AccountId` is best as we're moving in that direction - it's just an alias for `string` anyway